scripts/development/m3_stack/start.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -xe
+
+echo "Bringing up nodes in the backgorund with docker compose, remember to run ./stop.sh when done"
+docker-compose -f docker-compose.yml up -d --renew-anon-volumes
+echo "Nodes online"
+
+echo "Initializing namespace"
+curl -vvvsSf -X POST localhost:7201/api/v1/namespace -d '{
+  "name": "prometheus_metrics",
+  "options": {
+    "bootstrapEnabled": true,
+    "flushEnabled": true,
+    "writesToCommitLog": true,
+    "cleanupEnabled": true,
+    "snapshotEnabled": true,
+    "repairEnabled": false,
+    "retentionOptions": {
+      "retentionPeriodNanos": 172800000000000,
+      "blockSizeNanos": 7200000000000,
+      "bufferFutureNanos": 600000000000,
+      "bufferPastNanos": 600000000000,
+      "blockDataExpiry": true,
+      "blockDataExpiryAfterNotAccessPeriodNanos": 300000000000
+    },
+    "indexOptions": {
+      "enabled": true,
+      "blockSizeNanos": 7200000000000
+    }
+  }
+}'
+echo "Done initializing namespace"
```

Comment:
nit: steal the validation that paul does in the integration test - i.e. the `curl ... | jq ...` bits

Reply:
done